src/applications/applications.controller.ts

```
@@ -1,3 +1,15 @@
+  Body,
+  Controller,
+  Post,
+  Request,
+  UseGuards,
+  ConflictException,
+  Get,
+  Param,
+  ParseIntPipe,
+  Req,
+} from '@nestjs/common';
 import { Body, Controller, Post, Request, UseGuards, ConflictException, Get, Param, ParseIntPipe, Req, Patch } from '@nestjs/common';
```

Biome (1.9.4)

[error] Shouldn't redeclare 'Body'. Consider to delete it or rename it.

'Body' is defined here:


(lint/suspicious/noRedeclare)